internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -100,7 +105,12 @@ func (opts *Opts) Run() error {
 		return err
 	}
 
-	fmt.Println("Creating your cluster...")
+	runMongoShell, err := opts.askMongoShellQuestion()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Creating your cluster... [Press 'Ctrl + C' to stop waiting]")
```

Comment:
[nit] first, thanks for adding this, is something I wanted to add eventually, one small nit on copy to make this feel less urgent
```suggestion
	fmt.Println("Creating your cluster... [It's safe to 'Ctrl + C'")
```
Also I believe you don't show the generated password until the end, if I stop this I may lose that password so it may require a bit more than just adding this line (also why I have not voiced too strongly about it)

Reply:
this is the shown message when the user uses `Ctrl + c`

```
Creating your cluster... [Press 'Ctrl + C' to stop waiting]
/ ^C
You can connect to your Atlas cluster with the following user:
username:zxzxzxzx
password:cFBbSE9IU0d0
```

Reply:
perfect